doc/source/reference/random/bit_generators/index.rst

```diff
@@ -107,6 +107,6 @@ convenient ways.
    :toctree: generated/
 
     SeedSequence
-    bit_generator.ISeedSequence
-    bit_generator.ISpawnableSeedSequence
-    bit_generator.SeedlessSeedSequence
+    _bit_generator.ISeedSequence
```

Comment:
This looks wrong. You should never have something starting with an underscore in a toctree listing (or anywhere in the docs really).

So when I asked if there was anything in `bit_generator` besides `SeedSequence` that needs to be public, I guess the right answer was not "no" but "`ISeedSequence`, `ISpawnableSeedSequence` and `SeedlessSeedSequence`" (these are needed for writing other bitgenerators outside of NumPy, right?)?

Reply:
No, they are needed to implement new SeedSequence interfaces as kind of an abstract base class. BitGenerators should use `bit_generator.SeedSequence`. I think we should remove these then, and maybe document them in the (as yet nonexistant) random C-API documentation, not in the random python reference documentation.

Reply:
If they're not supposed to be used from Python at all, but only from Cython/C for new BitGenerator authors, then that sounds like the right course of action.

Reply:
What should we do with text like [this](https://numpy.org/devdocs/reference/random/bit_generators/mt19937.html#numpy.random.MT19937) "One may also pass in an implementor of the ISeedSequence interface like SeedSequence"? How about simplifying it to not mention ISeedSequence ?

Reply:
adopted the "don't mention" approach for now.

Reply:
that sounds good to me